doc/release-notes/release-8.sgml.in

```diff
@@ -30,9 +30,19 @@ The Squid-@SQUID_RELEASE@ change history can be <url url="https://github.com/squ
 
 <p>The most important of these new features are:
 <itemize>
-	<item>
+	<item>Helper binary changes
 </itemize>
 
+<sect1>Helper binary changes
```

Comment:
Please do not use the phrase "helper binary" in this context. These changes are not really about the changes in a binary file. If another helper (that happens to be a script rather than a binary) changes, it would presumably be documented in this subsection as well -- we are not going to add a "Helper script changes".

```suggestion
<sect1>Helper changes
```